File: src/main/java/pl/reaper/container/jython/scripts/Output.java
package pl.reaper.container.jython.scripts;

public class Output {

    private String result = "";

    public String getResult() {
        return result;
    }

    public void setResult(String result) {
        this.result = result;
    }
    
    public void appendResult(String result) {
        this.result += result;
    }
}

File: src/main/resources/scripts/sources/base/output.py
class Output:
  _result = ''
  
  def __init__(self):
      global outputAsString
      self._result = outputAsString

  def getResult(self):
    tmp = self._result
    self._result = ''
    return tmp

  def setResult(self, result):
    self._result = result

  def appendResult(self, result):
    self._result += result
output = Output()